src/pudl/metadata/resources/glue.py

```diff
@@ -87,6 +87,148 @@
         "etl_group": "glue",
         "sources": ["epacamd_eia"],
     },
+    # "ferc1_eia": {
+    #     "description": """This table connects FERC1 plant tables to EIA's plant-parts
+    # via record linkage. It is reported
+    # FERC plant records are reported... kind of messily. In the same table there are records
+    # that are reported as whole plants, generators, collections of prime movers. So we have
+    # this heterogeneously reported collection of parts of plants in FERC1.
+    # EIA on the other hand is reported in a much cleaner way. The are generators with ids and
+    # plants with ids reported in *seperate* tables. What a joy. In plant_parts_eia, we've
+    # generated the EIA plant-parts. The EIA plant-parts generated records for various levels
+    # or granularities of plant parts.
+    # For each of the FERC1 plant records we want to figure out which EIA plant-parts record
+    # is the corresponding record.
```

Comment:
I think we should work on this description. It'll probably be long but this is a valuable and also complex table.

Reply:
halp